Assets/GothicVR/Scripts/Phoenix/Util/ChangeLevelTriggerHandler.cs

```diff
@@ -0,0 +1,20 @@
+using GVR.Manager;
+using UnityEngine;
+
+namespace GVR.Phoenix.Util
+{
+    public class ChangeLevelTriggerHandler : MonoBehaviour
+    {
+        public string levelName;
+        public string startVob;
+        
+        void OnTriggerEnter(Collider other)
+        {
+            if (!other.CompareTag("Player"))
+                return;
+            
+            GvrSceneManager.I.LoadWorld(levelName.Split(".")[0], startVob.Trim());
```

Comment:
Why can't we just say the full name? Here we strip .zen and inside the method we add it again. I would recommend just leave it as it is.

Reply:
Done, had to modify each scene and folder name for occlusion, but it works